.github/workflows/mkdocs-build.yml

```diff
@@ -16,6 +16,11 @@ jobs:
         with:
```

Comment:
Qt3D in Qt6 features a better rendering, performance improvements and support for
modern graphics APIs like Vulkan, Metal and DirectX 12 that are all very explicit APIs
that expose more direct control over the GPU than OpenGL does.
See https://www.qt.io/blog/what-about-qt-3d-in-qt-6 for more details.

Moreover, migrating to new major versions can always be difficult and doing it is useful
to do not rely on deprecated versions and speed up migrations to future minor versions
in case of useful new features.

Commit message updated with same text.

Reply:
Top ! Thx